components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/util/IdPManagementConstants.java

```diff
@@ -74,6 +74,11 @@
     public static final String PROVISIONING =
             "INBOUND_PROV_ENABLED, INBOUND_PROV_USER_STORE_ID, DEFAULT_PRO_CONNECTOR_NAME";
 
+    //Default SAML endpoints.
+    public static final String SSO_ARTIFACT_URL = "/samlartresolve";
+    public static final String SAML_ECP_URL = "/samlecp";
+    public static final String SAMLSSO = "/samlsso";
```

Comment:
When giving a path should we really give '/', doesn't the builder handles it.
Even if we give it should handle IMO

Reply:
No we don't need to give "/". Builder can handle it.